.devcontainer/README.md

```diff
@@ -1,8 +1,24 @@
 # :package: FIMS devcontainer cheat sheet
 
-This cheat sheet outlines the steps for setting up a consistent FIMS development environment on a Windows machine using Windows Subsystem for Linux 2 (WSL2), Docker Engine, and Visual Studio Code (VS Code). By using a Docker image, this approach bypasses manual dependency installation and keeps everyone's setup the same.
+This cheat sheet outlines the steps for setting up a consistent FIMS environment.
+The approach bypasses manual dependency installation and keeps everyone's setup the same.
+Please follow the section that matches how you plan to open the container.
 
-## :rocket: Launch the FIMS environment
+- :globe_with_meridians: Open the container using a Codescpace from the browser
+- :technologist: Open the container on a Windows machine using Windows Subsystem
+  for Linux 2 (WSL2), Docker Engine, and Visual Studio Code (VS Code)
```

Comment:
I prefer the list of options be included in the sentence rather than as bullets. For example,
> Please follow the section below that matches how you plan to open the container, i.e., follow the [:globe_with_meridians: browser section](#-globe_with_meridians-browser) if you wish to open the container using Codespace from your favorite web browser and [:technologist: local section](#-technologist-local) if you wish to launch the container on your own machine.

I used hyperlinks so users can just click on the appropriate section for them and be taken to the correct location. Also, if you choose to leave this as is, there is a typo in Codespace on line 7.